allennlp_models/generation/models/copynet_seq2seq.py

```diff
@@ -76,8 +75,7 @@ def __init__(
         source_embedder: TextFieldEmbedder,
         encoder: Seq2SeqEncoder,
         attention: Attention,
-        beam_size: int,
-        max_decoding_steps: int,
```

Comment:
It does make sense to remove these, but I'd like to keep backwards compatibility. So let's check for these parameters in `**kwargs`. If we find them in `**kwargs`, issue a `DeprecationWarning` and pass them through to `beam_search.construct()`.

Reply:
Good call, I agree. I addressed this in `copynet_seq2seq.py` and added a test in `test_copynet_seq2seq.py`. Mind taking a look? If you are happy with it I will do it for the rest of the models!